examples/js/loaders/GLTFLoader.js

```diff
@@ -166,7 +188,7 @@ THREE.GLTFLoader = ( function () {
 
 			var parser = new GLTFParser( json, extensions, {
 
-				path: path || this.path || '',
+				path: path || this.resourcePath || '',
```

Comment:
Maybe we should remove the `path` argument to `parse()`... so instead of `parse(content, path, onLoad, ...)` it would be `parse(content, onLoad, ...)`. Especially if we're making a breaking change in this same method's signature at #14815.

Reply:
If we do that, doing in another PR seems better? This PR should be in r97 for the loader's consistent behavior but probably #14815 won't be in r97 because of lack of time.

Reply:
Sure. 👍